src/kernel/src/fs/vnode.rs

```diff
@@ -128,10 +129,13 @@ impl Drop for Vnode {
 }
 
 /// An implementation of `vtype`.
-#[derive(Debug)]
+#[derive(Debug, Clone, Copy)]
```

Comment:
`VnodeType` should not implement `Copy` because we embed additional data in the variants.